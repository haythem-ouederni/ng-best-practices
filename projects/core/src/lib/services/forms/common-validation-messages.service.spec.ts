import {async, TestBed} from '@angular/core/testing';
import {CommonValidationMessagesService} from './common-validation-messages.service';

describe('CommonValidationMessages', () => {
  let service: CommonValidationMessagesService;
  beforeEach(async(() => {
    TestBed.configureTestingModule({});
  }));

  beforeEach(() => {
    service = TestBed.get(CommonValidationMessagesService);
  });

  test('should be created', () => {
    expect(service).toBeTruthy();
  });

  test('should match snapshot', () => {
    expect(service).toMatchSnapshot();
  });

  test('#validationMessages should match snapshot', () => {
    expect(service.validationMessages).toMatchSnapshot();
  });
});
