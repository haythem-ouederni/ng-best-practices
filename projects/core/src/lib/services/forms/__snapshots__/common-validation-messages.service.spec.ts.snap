// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonValidationMessages #validationMessages should match snapshot 1`] = `
Object {
  "invalid": Object {
    "pattern": "Invalid input",
  },
  "length": Object {
    "maxlength": Object {
      "maxlength": [Function],
    },
    "minlength": Object {
      "minlength": [Function],
    },
  },
  "numbers": Object {
    "decimal": Object {
      "pattern": Object {
        "pattern": "Should be a number",
      },
    },
    "integer": Object {
      "pattern": Object {
        "pattern": "Should be an integer",
      },
    },
    "max": Object {
      "max": [Function],
    },
    "min": Object {
      "min": [Function],
    },
  },
  "required": Object {
    "required": "Required input",
  },
}
`;

exports[`CommonValidationMessages should match snapshot 1`] = `CommonValidationMessagesService {}`;
